src/SFA.DAS.Rofjaa.Database/Scripts/AgencyImportTest.sql

```diff
@@ -1,2 +1,2 @@
-﻿INSERT INTO [dbo].[Agency] VALUES (1, 1);
-INSERT INTO [dbo].[Agency] VALUES (2, 0);
\ No newline at end of file
+﻿INSERT INTO [dbo].[Agency] (LegalEntityId, IsGrantFunded, EffectiveFrom, EffectiveTo) VALUES (1, 1, '2020/01/01', '2025/01/01');
```

Comment:
Technically, this script won't work without the EffectiveTo field, I expect

Reply:
Good spot! Removed and committed